src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -913,7 +948,7 @@ private function registerSerializerConfiguration(array $config, ContainerBuilder
             // Run after serializer.normalizer.object
             $definition = $container->register('serializer.normalizer.data_uri', DataUriNormalizer::class);
             $definition->setPublic(false);
-            $definition->addTag('serializer.normalizer', ['priority' => -920]);
+            $definition->addTag('serializer.normalizer', array('priority' => -920));
```

Comment:
good catch (my bad), it should probably be moved to another PR targeting the 3.1 branch.